docs/upgrade.html

```diff
@@ -61,6 +61,25 @@ <h4><a id="upgrade_3_4_0" href="#upgrade_3_4_0">Upgrading to 3.4.0 from any vers
         </li>
     </ol>
 
+<h4><a id="upgrade_kraft_3_4_0" href="#upgrade_kraft_3_4_0">Upgrading a KRaft-based cluster to 3.4.0 from any version 3.0.x through 3.3.x</a></h4>
+
+    <p><b>If you are upgrading from a version prior to 3.4.0, please see the note below. Once you have changed the metadata.version to the latest version, it will not be possible to downgrade to a version prior to 3.4-IV0.</b></p>
+
```

Comment:
I think this part should say "upgrading from a version prior to 3.3.0"  and "downgrade to a version prior to 3.3-IV0".